net/znc/Makefile

```diff
@@ -23,6 +23,7 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS := CONFIG_ZNC_ICU
 
```

Comment:
That symbol does not conrol the visibility of menu items, it tells the buildsystem to rebuild znc when its value changes.